dashboard/data/combine.py

```diff
@@ -108,7 +110,7 @@ def combine_bmg_echo_data(
     echo_df: pd.DataFrame,
     df_stats: pd.DataFrame,
     plate_values: np.ndarray,
-    modes: dict[Mode],
+    modes: dict[Mode] = dict(),
```

Comment:
mutable objects shouldn't be set as defaults in functions as this introduces a subtle bug in form of them being shared among all function calls in the program

instead if you want to set an empty dict as default you should write:

```python
def func(
    ...
    should_default_to_dict: dict | None = None,
    ...
) -> ...:
    if should_default_to_dict is None:
        should_default_to_dict = {}
    ...
```

this way each function call creates its own dictionary